product_docs/docs/pgd/5/manual/07-configure-proxies.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Step 7 - Configure proxies
+navTitle: Configure proxies
+deepToC: true
+---
+
+## Configure proxies
+
+### Configure the cluster for proxies
+
+To set up a proxy, you will need to first prepare the cluster and sub-group the proxies will be working with by:
+
+* Logging in and setting the `enable_raft` and `enable_proxy_routing` node group options to `true` for the sub-group. Use [`bdr.alter_node_group_option`](/pgd/latest/reference/nodes-management-interfaces#bdralter_node_group_option), passing the sub-group name, option name and new value as parameters.
+* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to.
+* Create a pgdproxy user on the cluster with a password (or other authentication)
+
+### Configure each host as a proxy
+
+Once the cluster is ready, you will need to configure each host to run pgd-proxy by:
+
+* Creating a pgdproxy local user
+* Creating a `.pgpass` file for that user which will allow it to log into the cluster as `pgdproxy`.
+* Modify the systemd service file for pgdproxy to use the pgdproxy user.
+* Create a proxy config file for the host which lists the connection strings for all the nodes in the sub-group, specifies the name that the proxy should use when connected and gives the endpoint connection string the proxy will accept connections on.
+* Install that file as `/etc/edb/pgd-proxy/pgd-proxy-config.yml`
+* Restart the systemd service and check its status.
+
+Further detail on all these steps is included in the worked example.
+
+## Worked example
+
+## Preparing for proxies
+
+For proxies to function, the `dc1` subgroup must enable Raft and routing.
+
+Log into any node in the cluster.
+
+```
+SELECT bdr.alter_node_group_option('dc1', 'enable_raft', 'true');
+SELECT bdr.alter_node_group_option('dc1', 'enable_proxy_routing', 'true');
+```
+
+Next, create a PGD proxy within the cluster using the `bdr.create_proxy` function.
+This function takes two parameters, the proxies unique name and the group it should be a proxy for.
+
+In our example, we want a proxy on each host in the dc1 sub-group:
+
+```
+SELECT bdr.create_proxy('pgd-proxy-one','dc1');
+SELECT bdr.create_proxy('pgd-proxy-two','dc1');
+SELECT bdr.create_proxy('pgd-proxy-three','dc1');
+```
+
```

Comment:
`bdr.create_proxy()` also returns `void` (and nothing else) on success, so creation can be confirmed with e.g.:

---
The [`bdr.proxy_config_summary`] (catalogs-internal#bdrproxy_config_summary) view can be used to check that the proxies were created:

```
postgres=# SELECT proxy_name, node_group_name
             FROM bdr.proxy_config_summary;
   proxy_name    | node_group_name
-----------------+-----------------
 pgd-proxy-one   | dc1
 pgd-proxy-two | dc1
 pgd-proxy-three | dc1
(3 rows)
```

Reply:
Done